python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/slice_conditions.py

```diff
@@ -48,6 +48,31 @@ def compute_slice(self, context: SchedulingContext) -> AssetSlice:
         return context.asset_graph_view.compute_in_progress_asset_slice(context.asset_key)
 
 
+@whitelist_for_serdes
+class WillBeRequestedCondition(SliceSchedulingCondition):
```

Comment:
nit: I think class name should include the word "tick" just like the static constructor and description